smallrye-reactive-messaging-cloud-events/src/main/java/io/smallrye/reactive/messaging/cloudevents/DefaultCloudEventMessage.java

```diff
@@ -61,7 +62,13 @@ public String getId() {
   }
 
   @Override
+  @JsonIgnore
   public T getPayload() {
     return delegate.getData().orElseThrow(() -> new IllegalArgumentException("Invalid message - no payload"));
   }
+
+  @Override
+  public String toString() {
+    return delegate.toString();
```

Comment:
Any reason for this?

Reply:
hrm... I personally like "proper" toString :-)   I can remove it if you dont want it